.gitignore

```diff
@@ -1,3 +1,6 @@
 /target/
 Cargo.lock
 *~
+**/*.rs.bk
```

Comment:
A simple `*.rs.bk` should be enough -- like the `*~` pattern above.

Reply:
Yeah, you're right.

Reply:
Well, that's just a pattern I took from `.gitignore`, autogenerated by `cargo new foo` so I didn't give it a second thought.